gwt-uibinder-client/src/main/java/org/gwtproject/uibinder/client/UiBinder.java

```diff
@@ -15,8 +15,6 @@
  */
```

Comment:
a few of these have 2008 date and is formatted differently than the others.

Reply:
The format was consistent enough that checkstyle took it - it doesnt care about year (and neither does Google, since upstream gwt is full of varying years).